docs/reference/errors-and-warnings/NU5036.md

```diff
@@ -0,0 +1,27 @@
+---
+title: NuGet Error NU5036
+description: NU5036 Error code
+author: rrelyea
+ms.author: karann
+ms.date: 8/8/2018
+ms.topic: reference
+ms.reviewer: anangaur
+f1_keywords: 
+  - NU5036
+---
+
+# NuGet Error NU5036
+<pre>This package has an improperly escaped Url in LicenseUrl</pre>
+
+### Issue
+
+LicenseUrl metadata value in the nuspec file should be properly escaped.
+Some versions of nuget pack functionality have a problem which will be updated asap.
+
+### Solution
+
+Use a fixed version of nuget pack functionality:
+* "NuGet.exe pack" - fixed in 4.9.1
```

Comment:
4.9.1 is still not available. So lets mention that is broken in 4.9.0 and fix will be available in 4.9.1